### PR TITLE
feat: add searchScore field to pages

### PR DESCRIPTION
**Pull request by dopry**
Add a searchScore property to the pages types.

### CODE REVIEW

File: example/example/tests/test_grapple.py [hunk 1]
```diff
@@ -442,24 +442,61 @@ def setUpTestData(cls):
...
+        self.assertEquals(page_data[2]["searchScore"], None)
         self.assertEquals(page_data[3]["title"], "Alpha Gamma")
+        self.assertEquals(page_data[3]["searchScore"], None)
         self.assertEquals(page_data[4]["title"], "Beta Alpha")
+        self.assertEquals(page_data[4]["searchScore"], None)
         self.assertEquals(page_data[5]["title"], "Gamma Alpha")
+        self.assertEquals(page_data[5]["searchScore"], None)
+
+    @unittest.skipIf(
+        connection.vendor != "postgresql",
```
[the remark this answers is not in the document]
Comment (dopry) [reply example/example/tests/test_grapple.py:472]: I'm not running mysql, so I don't know if the test would pass or fail or what search scores would be generated by it. The same with elastic search. My instinct says each search backend is going to return different scores and ordering. I feel like we should implement tests for each search backend as we implement testing support for that backend. Although setting this up to fail when a new search backend is added to our test matrix, would probably help ensure we don't overlook it when we add those.  I was just focusing on the postgres case here. How do you think we should proceed?

[the remark this answers is not in the document]
Comment (dopry) [reply example/example/tests/test_grapple.py:472]: The more I thought about it, I agree with the test selection solely on sqlite. Although at some point in the future the test selection will need to consider both the search backend and database connection.

[the remark this answers is not in the document]
Comment (dopry) [reply example/example/tests/test_grapple.py]: @zerolab  any other feedback on this, or can we work on getting it merged?

**Review by zerolab — APPROVED** [review]

**Comment by dopry** [comment]
🎉

**Pull request merged**